### ISSUE TITLE
Character Crash before versus screen

### ISSUE THREAD

**Issue by DarkerFalcon** [ikemen-engine/Ikemen-GO#653]
When i try to select Popeye, I.K.E.M.E.N. crashes before versus screen with an error.
[Ikemen.log](https://github.com/ikemen-engine/Ikemen-GO/files/9409641/Ikemen.log)
Idk how to fix this. btw i use Windows 11, the character is PopeyeX by DDR (Derrick D. Rowell). If someone can help me. he'll give me a huge help. The version of IKEMEN is 0.98.2 (the latest one)

**Comment by lokpro** [comment]
popeyeX.cns  line 25810:
```
endcmdbuftime =; (winmugen only)
```
empty value causes the error.

Remove this line to play it in ikemenGo.

Mugen 1.1 doesn't show the error.

**Comment by lokpro** [comment]
https://github.com/ikemen-engine/Ikemen-GO/blob/f4abe2fbfe2ce97bc1b366c2adb6a80bb4c98d9b/src/compiler.go#L1158-L1160

**Comment by lokpro** [comment]
Found another case.
Superman By Orochi_shin
https://discord.com/channels/233345562261323776/282927929548210177/1019846098979917824

**Comment by PGamer3** [comment]
I also have the same error, only this time with Opera Elmer X.

**Comment by PGamer3** [comment]
There is also the same error with Opera Elmer X
[Ikemen.log](https://github.com/ikemen-engine/Ikemen-GO/files/11644226/Ikemen.log)

Does anyone have solutions?

**Comment by potsmugen** [comment]
> Perhaps it's better to change to ignore the line instead of throwing an error.

I disagree. The compiler already tells people where and what the error is so they can fix/erase the line themselves.

**Comment by waspennator** [comment]
Decided to track down and retest with the latest nightly, from what it seems PopeyeX and Superman by Orochi Shin seem to load fine for me at least without crashing in Ikemen GO nightly with this. But ElmerX still crashes ikemen go. Again he loads up perfectly fine on 1.1b

[0Elmer_DDR_99_9X.zip](https://github.com/user-attachments/files/31383496/0Elmer_DDR_99_9X.zip)

<img width="394" height="294" alt="Image" src="https://github.com/user-attachments/assets/f2df210b-2fc0-4691-93f3-b5945c460ee1" />

Edit: Also seems like the original Elmer version by DDR works fine.

**Comment by potsmugen** [comment]
Some of these just seem completely unrelated. The latest one has this expression:
`trigger1 =  stateno = 11 stateno = 0 ||stateno = 100 ||`

It's always true in Mugen. Not even sure what to make of that.

Some test results:
`stateno = 11 stateno = 0 ||stateno = 100 ||` -> Compiles. Seems to always evaluate to true
`stateno = 11 stateno = 0 ||` -> Crashes
`||stateno = 100 ||` -> Crashes
`stateno = 11 stateno = 0` -> Crashes

Looks like they found the only way to write it without crashing.

Update: So after wasting some time trying to find a pattern in that mess, I did find one but as expected it didn't make any sense. It only happens when you break the expression in two specific ways, and each of those ways already make Mugen crash on their own.

A change that would make Ikemen worse just by existing, even if gated by version check.
